.github/renovate.json5

```diff
@@ -154,6 +154,12 @@
       pinVersions: true,
       automerge: true,
     },
+    {
+      matchPackageNames: [
+        "sigs.k8s.io/e2e-framework",
```

Comment:
how will we notice the new version is out? we could disable automerge, as well, but that would mean we have to keep the PR open for a while...

Reply:
thats the point @0sewa0 and me discussed, we thought that not having this up to date for a while is not a problem, since the e2e tests are running anyway also on an older version. But we can also just disable auto-update i am also fine with this @0sewa0 your take on this?

Reply:
the e2e framework has a single purpose, run our e2e tests reliably, if they do that on the current version, then I see 0 reason for upgrading
- we don't have to worry about security vulnerabilities (IMO the MAIN reason we update our dependencies), as it is not production code
- if the new version of the framework has "cool new stuff" in it, getting the new version via renovate will not let us know about the "cool new stuff" and it's value, somebody will have to search/discover it on their own anyway.
   - Sidenote: most e2e tests I saw written tend to be "copy-pasta" from other e2e tests, so I don't see people here being adventurous and searching for new ways to use the framework 😅 🙈

Reply:
Can we just disable auto merge for this specific package?

Reply:
but i'm agree we can ignore for now upgrading since it's just for e2e tests.